docs/how-to/upgrade-cloud.rst

```diff
@@ -122,13 +122,13 @@ allows **COU** to be used by scripts or during upgrade testing. A quiet mode swi
 is also offered, which will only output error logs and a completion message to STDOUT.
 
 Usage examples
-^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~
 
 Non-interactive mode:
 
-.. code:: 
+.. terminal:: 
+    :input: cou upgrade --no-interactive
```

Comment:
this needs to be updated

Reply:
(we no longer have a `--no-interactive` option)

Reply:
Ah right, I forgot about this CLI change. Thanks for pointing it out.